routes/web.php

```diff
@@ -160,5 +161,5 @@
 Route::get('event-sourcing', [EventSourcingController::class, 'show']);
 Route::post('event-sourcing', [EventSourcingController::class, 'subscribe']);
 
-
+Route::get('technologies', [TechnologiesController::class, 'index']);
```

Comment:
I would rename this to `/uses` (seems like that is the default for this kind of content)